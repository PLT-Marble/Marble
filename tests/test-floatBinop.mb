main(){
    printf(1.1+2.2);
    printf(1.1*2.2);
    printf(1.1-2.2);
    printf(1.1/2.2);
    return 0;
}