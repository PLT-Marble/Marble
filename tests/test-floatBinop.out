3.3
2.42
-1.1
0.5
